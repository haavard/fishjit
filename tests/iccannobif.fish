f10!/{:}0=?v/{1-}}:{+:0}!/a,:1%-:0=?!v~{:1!/$:@0=?!v$~}}:&{{&*\
 /-$/    ;n/\oa    n:    \       }+1{/     \$-1$*a /|.!22}}01@/
 * :{:}(?v:{!":}-1!/$:@0=?!v$~{:}1!/$:@0=?!v$~}}}:&{{{&*:1%-*&{{&+}}{1+}2:.
 b .1 +cb   \      \$-1$*a /       \$-1$,a /
 \*9{~~~{/
