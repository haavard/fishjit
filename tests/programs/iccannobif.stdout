0
1
1
2
3
5
8
31.000000
93.000000
421.000000
415.000000
638.000000
3501.000000
9314.000000
51821.000000
53116.000000